webExtensionApis/study/schema.json

```diff
@@ -1,73 +1,154 @@
 [
   {
     "namespace": "study",
-    "description": "study",
+    "description":
+      "Interface for shield studies, for use by `background.js` scripts.",
+    "apiVersion": 5,
+    "types": [
+      {
+        "name": "studySetup"
+      },
+      {
+        "name": "weightedVariations"
+      },
+      {
+        "name": "anEnding"
```

Comment:
I don't see schemas inside `.webExtensionApis/study/schemas` for `anEnding`, `telemetrySelectOptions` and `prefTypes` -- should there be?

Reply:
It will get there.